exporters/jaeger/uploader.go

```diff
@@ -112,8 +115,15 @@ func WithAgentPort(port string) AgentEndpointOption {
 	})
 }
 
+var emptyLogger = logr.Logger{}
+
 // WithLogger sets a logger to be used by agent client.
 func WithLogger(logger *log.Logger) AgentEndpointOption {
```

Comment:
We should have some comments that `WithLogger` and `WithLogr` will overwrite each other.